### PR TITLE
LF-3585: Create custom revenue types tile view

### PR DESCRIPTION
**Pull request by SayakaOno**
**Description**

Create the custom revenue types tile view by following the same pattern for the expense types.
- create `useSortedCustomRevenueTypes` hook
- create `ManageCustomRevenueTypes` container
- fix the readonly custom expense path (Thank you for your [comment](https://github.com/LiteFarmOrg/LiteFarm/pull/2859#discussion_r1329283248) Duncan!)
- add CSS to scss file for `Tiles` to bold Icon tile label
   - update some files imports to use it
   - delete one scss file

Jira link: https://lite-farm.atlassian.net/browse/LF-3585

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

- [ ] Passes test case
- [x] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [x] I have added "MISSING" for all new language tags to languages I don't speak
- [x] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

File: packages/webapp/src/containers/Finances/ManageCustomExpenseTypes/index.jsx [hunk 1]
```diff
@@ -18,13 +18,13 @@ import { useTranslation } from 'react-i18next';
...
 import { icons } from '../NewExpense/ExpenseCategories';
-import labelIconStyles from '../NewExpense/ExpenseCategories/styles.module.scss';
+import labelIconStyles from '../../../components/Tile/styles.module.scss';
 import useCustomExpenseTypeTileContents from '../useCustomExpenseTypeTileContents';
 
 const addCustomTypePath = '/add_custom_expense';
 
 const getPaths = (typeId) => ({
-  readOnly: `/readOnly_custom_expense/${typeId}`,
+  readOnly: `/readonly_custom_expense/${typeId}`,
```
Comment (Duncan-Brain) [inline packages/webapp/src/containers/Finances/ManageCustomExpenseTypes/index.jsx:27]: 🍕

**Review by Duncan-Brain — APPROVED** [review]
⚡

**Review by kathyavini — APPROVED** [review]
Looks great!

Only thing is (and this is my fault!!) I never added the `history.push` to this container in the sagas (it was a long time ago and this component was not yet on the horizon -- although in retrospect I should have asked you for the likely name and added anyway 🙂 )

Do you want to add those `history.push('manage_custom_revenues')` to this ticket? Otherwise I'll open a new PR using the saga ticket number.

**Comment by SayakaOno** [comment]
@kathyavini 
Thank you Joyce! Since both you and Duncan have approved, could you open a new PR?

**Pull request merged**